lib/SIL/TypeLowering.cpp

```diff
@@ -1034,17 +1027,6 @@ namespace {
       llvm_unreachable("cannot destroy an UnsafeValueBuffer!");
     }
   };
-  
-  /// A class that acts as a stand-in for improperly recursive types.
-  class RecursiveErrorTypeLowering : public AddressOnlyTypeLowering {
-  public:
-    RecursiveErrorTypeLowering(SILType type)
-      : AddressOnlyTypeLowering(type) {}
-    
-    bool isValid() const override {
```

Comment:
One more nit-pick if you don't mind: It looks like TypeLowering::isValid() is never called anywhere and always returns true now, so it can be removed?